addons/website_forum/views/website_forum.xml

```diff
@@ -218,6 +226,19 @@
     </t>
 </template>
 
+<template id="website.403">
```

Comment:
Don't overwrite the 403 template if it's to erase it. 

Create another one instead.